magicctapipe/image/calib.py

```diff
@@ -0,0 +1,102 @@
+
+
+import numpy as np
+
+
+from ctapipe.image import (
+    apply_time_delta_cleaning,   
+    number_of_islands,
+    tailcuts_clean,   
+)
+
+from lstchain.image.cleaning import apply_dynamic_cleaning
+from lstchain.image.modifier import (
+    add_noise_in_pixels,
+    random_psf_smearer,    
+)
+
+__all__ = [
+    "Calibrate_LST", "Calibrate_MAGIC"
+]
+
+def Calibrate_LST(event, tel_id, rng, config_lst, camera_geoms, calibrator_lst, increase_nsb, use_time_delta_cleaning, use_dynamic_cleaning ):
+
+    """
+    This function computes and returns signal_pixels, image, and peak_time for LST
+    """
```

Comment:
Docstring is a bit lacking. Function returns the quantities for a single event of a telescope of  LST type. 
Input parameters are not explained. Output format are not described.

Reply:
Fixed (?)

Reply:
It looks ok. Not a fan of the "some information" and the difference of format for giving the type of inputs vs output.
But I could accept this.

Reply:
Changed a bit